#!/usr/bin/env python

PACKAGE="gazebo_sfm_plugin"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pedestrian_velocity", double_t,0, "Velocidad peaton",.5, 0,1)

exit(gen.generate(PACKAGE, "PedestrianSFMPlugin", "PedestrianSFMPlugin"))